packages/create-contentful-app/template/README.md

```diff
@@ -0,0 +1,62 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `yarn configure`
+
+This will help you configure your development environment, it will log into
+contentful if necessary, and then help you choose the space and environment you
+wish to use with your app.
+
+
+### `yarn start`
+
+Creates or updatse your app definition in contentful, and runs the app in the 
+development mode.
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
```

Comment:
Why would I see it in the browser? It only make sense to open it from the Web App

Reply:
Probably left over docs from CRA